vck/src/commonTest/kotlin/at/asitplus/wallet/lib/data/rfc7519/AudienceSerializerTest.kt

```diff
@@ -0,0 +1,70 @@
+package at.asitplus.wallet.lib.data.rfc7519
+
+import at.asitplus.wallet.lib.data.rfc7519.primitives.Audience
+import at.asitplus.wallet.lib.data.rfc7519.primitives.StringOrUri
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.cbor.Cbor
+import kotlinx.serialization.decodeFromHexString
+import kotlinx.serialization.encodeToHexString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+
+@ExperimentalSerializationApi
+class AudienceSerializerTest : FreeSpec({
+    "simple tests" - {
+        val values = listOf(
+            listOf(StringOrUri("Test")),
+            StringOrUri("Test"),
+            "Test",
+            Audience(StringOrUri("Test")),
+            Audience(listOf(StringOrUri("Test"))),
+        )
+        withData(
+            data = values.mapIndexed { index, it ->
+                index.toString() to it
+            }.toMap()
+        ) { value ->
+            run {
```

Comment:
I really can't tell from looking at it ... what does this test? and why are there so many `run` blocks?

Reply:
removed